packages/core/src/modules/routing/RecipientModule.ts

```diff
@@ -47,6 +47,9 @@ export class RecipientModule {
   private mediationRepository: MediationRepository
   private routingService: RoutingService
 
+  // $stop is used for stop message pickup signal
+  private readonly stop$ = new Subject<boolean>()
```

Comment:
can we give this a more specific name? I'm already not a huge fan I named it stop$ for the generic agent shutdown subject

Reply:
I've set it to stopMessagePickup$ to be more specific. Didn't want it to collide with stopMessagePickup() method but, as it is a private field, it should not be confusing I think.